### DISCUSSION TITLE
Time offset in timeline UTC - local time

### DISCUSSION THREAD

**Discussion by DeltaTango69** [immich-app/immich#6674] [Feature Request]
Hello,

pictures from my Sony Camera has an offset to the pictures from the mobiles. The difference is the timezone. The only difference in the exif data are that in the mobil pictures are two entrys more.

![image](https://github.com/immich-app/immich/assets/82575543/f88bb548-6255-4302-8a0e-7273109df82f)

![image](https://github.com/immich-app/immich/assets/82575543/0b52447c-ab59-455c-9b2a-d291d0bf50cb)

The "offset" entrys are not in the Sony pictures. Could this be the reason? Other programms only use the information from the datetimeoriginal as Local - time. Than there is no offset in the timeline. 

Windows - time taken!

![image](https://github.com/immich-app/immich/assets/82575543/a905ca4e-817e-4d3f-8dce-dcc72456ef2c)

Immich - Info - one hour difference !

![image](https://github.com/immich-app/immich/assets/82575543/8fbf21e5-a2f1-4d69-8338-b4e64044da70)

Is it possible to use only the datetimeoriginal withou offset?

**Comment by DJPapa66** [comment]
Hello,

I have the same problem and don't understand why the recording date isn't just taken. The photo was taken at 15:52:37. That's it. Why does something have to be calculated? I'm "lucky" that it's only +1 hour for me. I could deal with that if necessary. However, I don't understand it.
It would be great if I could choose how to import.

**Comment by fwsmit** [comment]
I'm working on resolving the issues with timezone detection for DSLR pictures. At least on my Sony A7 first gen it's possible to infer the timezone from the exif metadata. To fix this issue for other camera's as well, I need some example pictures.

To submit example pictures, please follow the following steps:
- Set your camera to a time of 10:00 AM and a timezone of GMT+0 (No daylight savings time). Take the first picture.
- Set your camera to a timezone of GMT+8 (No daylight savings time). The time should now be 18:00. Take the second picture

Upload both pictures to the following github issue: https://github.com/photostructure/exiftool-vendored.js/issues/187. Please document clearly which image is which timezone and which camera you used.

**Comment by bo0tzz** [comment]
#6799
